README.md

```diff
@@ -1,57 +1,215 @@
 
-## Demo
+This document will guide you through the steps to deploy the solution to your environment.
 
-A demo app is included to show how to use the project.
+An Azure Active Directory (AAD) is required to register the app registrations. In this document, the AAD will be called "ETL AAD", and an account in the ETL AAD will be called ETL work account.
 
-To run the demo, follow these steps:
+* All app registrations are created in the ETL AAD. 
 
-(Add steps to start up the demo)
+An Azure Subscription is required to deploy the Azure components. The [ARM Template](azuredeploy.json) deploys these Azure components automatically. 
 
-1.
-2.
-3.
+## Build Docker Images
 
-## Resources
+   > **Note:** The following commands are running on your own computer.
 
-(Any additional resources or related projects)
+1. Clone the repository with Visual Studio Code.
 
-- Link to supporting information
-- Link to similar sample
-- ...
+2. Open **View > Integrated Terminal**.
+
+3. Execute the commands below to build and push the extracting image to the Docker Hub.
+
+   ```powershell
+   cd cmd/extracting
+   docker build -t YOURDOCKERACCOUNTNAME/extracting .
+   docker push YOURDOCKERACCOUNTNAME/extracting
+   ```
+
+4. Execute the commands below to build and push the transforming image to the Docker Hub.
+
+   ```powershell
+   cd ../extracting
+   docker build -t YOURDOCKERACCOUNTNAME/transforming .
+   docker push YOURDOCKERACCOUNTNAME/transforming
+   ```
+
+5. Execute the commands below to build and push the loading image to the Docker Hub.
+
+   ```powershell
+   cd ../extracting
+   docker build -t YOURDOCKERACCOUNTNAME/loading .
+   docker push YOURDOCKERACCOUNTNAME/loading
+   ```
+
+6. Execute the commands below to build and push the rendering image to the Docker Hub.
+
+   ```powershell
+   cd ../extracting
+   docker build -t YOURDOCKERACCOUNTNAME/rendering .
+   docker push YOURDOCKERACCOUNTNAME/rendering
+   ```
+
+## Create Azure Storage Account and File Share
+
+> **Note:** The following commands are running on Azure Portal.
+
+1. Open the **Shell** in the Azure Portal.
+
+   ![](images/deploy-01.png)
+
+2. Execute the command below to choose your subscription.
+
+   ```powershell
+   az account set --subscription SELECTED_SUBSCRIPTION_ID
+   ```
+
+3. Execute the commands below to create a new resource group.
+
+   > **Note:** Change the placeholder `[RESOURCE_GROUP_NAME]` to the name for the new resource group you will create.
+   
+   ```powershell
+   ACI_PERS_RESOURCE_GROUP=[RESOURCE_GROUP_NAME]
+   ACI_PERS_STORAGE_ACCOUNT_NAME=mystorageaccount$RANDOM
+   ACI_PERS_LOCATION=eastus
+   ACI_PERS_SHARE_NAME=acishare
+   az group create --location eastus --name $ACI_PERS_RESOURCE_GROUP
+   ```
+
+4. Execute the commands below to create the storage account.
+
+   ```powershell
```

Comment:
Change the powershell markdown to the appropriate actual scripting language (looks like bash for most)

Reply:
fixed